mlem/contrib/requirements.py

```diff
@@ -0,0 +1,52 @@
+"""Requirements support
+Extension type: build
+
+MlemBuilder implementation for `Requirements` which includes
+installable, conda, unix, custom, file etc. based requirements.
+"""
+import logging
+from typing import ClassVar, Optional
+
+from pydantic import validator
+
+from mlem.core.base import load_impl_ext
+from mlem.core.objects import MlemBuilder, MlemModel
+from mlem.core.requirements import Requirement
+from mlem.ui import EMOJI_OK, EMOJI_PACK, echo
+from mlem.utils.entrypoints import list_implementations
+
+REQUIREMENTS = "requirements.txt"
+
+logger = logging.getLogger(__name__)
+
+
+class RequirementsBuilder(MlemBuilder):
+    """MlemBuilder implementation for building requirements"""
+
+    type: ClassVar = "requirements"
+
+    target: Optional[str] = None
+    """Target path for requirements"""
+    req_type: str = "installable"
+    """Type of requirements, example: unix"""
+
+    @validator("req_type")
+    def get_req_type(cls, req_type):  # pylint: disable=no-self-argument
+        if req_type not in list_implementations(Requirement):
+            raise ValueError(
+                f"req_type {req_type} is not valid. Allowed options are: {list_implementations(Requirement)}"
+            )
+        return req_type
+
+    def build(self, obj: MlemModel):
+        req_type_cls = load_impl_ext(Requirement.abs_name, self.req_type)
+        assert issubclass(req_type_cls, Requirement)
+        reqs = obj.requirements.of_type(req_type_cls)
+        if self.target is None:
+            reqs_representation = [r.get_repr() for r in reqs]
+            requirement_string = "\n".join(reqs_representation)
```

Comment:
Maybe `" ".join`? Will it work like `pip install $(mlem build requirements -m model)` RN?

Reply:
I have replaced it with `" "`, but `pip install $(mlem build requirements -m model)` doesn't work because of the first line with the emoji i.e. `⏳️ Loading model from sk-model.mlem`.

Is there a way to ignore all this `echo` related stuff?

Reply:
Good question! This is a good example when we should try our tools in a user scenario to find out how it will work in fact :)

This is what I heard multiple times from Dmitry and Ivan, btw. See this https://github.com/iterative/mlem/issues/390 Let's discuss it there, so it won't get lost.

Reply:
We can add `--quiet/-q` to `mlem-callback` that will disable standard cli output

Reply:
Is this to be done? As a separate PR maybe? Or I need to add this `--quiet` stuff somewhere? I do see stuff related to `mlem_group_callback` so wondering if it is to be accommodated there?

Also, should this PR be merged since it's approved OR should we do the above first?